src/convert_image/sse2.rs

```diff
@@ -221,6 +223,30 @@ unsafe fn unpack_ui8x3_i16x2_4x(image: *const u8, sampler: Sampler) -> (__m128i,
     (_mm_or_si128(red, green), _mm_or_si128(blue, green))
 }
 
+#[inline(always)]
+unsafe fn rgb_to_bgra_lane_conversion(input: __m128i, output_buffer: *mut __m128i) {
+    let alpha_mask = _mm_set_epi32(0xff, 0xff, 0xff, 0xff);
+
+    // we have b3g3r3-- b2g2r2-- b1g1r1-- b0g0r0--
+    let aligned_line = _mm_unpacklo_epi64(
+        _mm_unpacklo_epi32(_mm_slli_si128(input, 1), _mm_srli_si128(input, 2)),
+        _mm_unpacklo_epi32(_mm_srli_si128(input, 5), _mm_srli_si128(input, 8)),
+    );
+
+    let res = _mm_or_si128(aligned_line, alpha_mask);
+
+    // Byte swap 128 bit
+    let shr = _mm_srli_epi16(res, 8);
+    let shl = _mm_slli_epi16(res, 8);
+    let ored = _mm_or_si128(shl, shr);
+    let pshuf32 = _mm_shuffle_epi32(ored, 0x4e);
```

Comment:
Can you make this explicit:
`let pshuf32 = _mm_shuffle_epi32(ored, mm_shuffle(1, 0, 3, 2));`

You can follow the same for the whole function